src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir.ttc) format("truetype"),
+    url(/fonts/Avenir-Book.ttf) format("truetype"),
+    url(/fonts/Avenir-Heavy.ttf) format("truetype");
+}
+/* #region body */
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)